tests/console/validate_resize_luks2_partition.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate resizing of an encrypted partition with luks2
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use Test::Assert ':all';
+
+sub run {
+    my $original_size       = get_var('HDD_SIZEGB');
+    my $encrypted_partition = get_var('AUTOYAST_PRE_ENCRYPT_DISK') . '1';
```

Comment:
I wonder if we should move it to `tests/console/validate_file_system.pm` instead of separate module.

Reply:
good idea, although I prefer not to mount the partition so I will do some adjustments there.